### PR TITLE
Droth 588 point like assets are not moved to new link geometry

### PR DESCRIPTION
**Pull request by MAC-CGI**
Droth-588 point like assets are not moved to new link geometry

### CODE REVIEW

File: digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/util/DataFixture.scala [hunk 1]
```diff
@@ -832,6 +832,31 @@ object DataFixture {
     println("\n")
   }
 
+  def testsVisualization(): Unit = {
+    val roadLinkService = new RoadLinkService(vvhClient, new DummyEventBus, new DummySerializer)
+    val municipalities: Seq[Int] = Seq(5)
+    //    val municipalities: Seq[Int] =
```
Comment (joanacoelho) [inline digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/util/DataFixture.scala]: Remove commented code

Comment (MAC-CGI) [inline digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/util/DataFixture.scala]: Thank you, this code in datafixture need to be deleted

**Comment by joanacoelho** [comment]
Ok to me

**Comment by oskarmalmstrom** [comment]
looks good to me

**Comment by halied** [comment]
Ok

File: digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/PointAssetFiller.scala [hunk 2]
```diff
@@ -42,27 +42,32 @@ object PointAssetFiller {
...
-         else if (changeInfo.oldEndMeasure.getOrElse (0.0) + MaxDistanceDiffAllowed >= mValue && changeInfo.oldEndMeasure.getOrElse (0.0) <= mValue)
-          correctGeometry (asset.id, newRoadLink, changeInfo.newEndMeasure.getOrElse (0.0), changeInfo.vvhTimeStamp)
-         else
-          correctValuesAndGeometry (asset, roadLinks, changeInfo, adjustmentOption)
-       case _ =>
-         None
-     }
+        }
+        val assetpoint = Point(asset.lon, asset.lat)
+        val roadOption = GeometryUtils.calculatePointFromLinearReference(newRoadLink.geometry, mValue)
```
Comment (marquesfm) [inline digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/PointAssetFiller.scala]: The variables assetpoint and roadOption can be removed.
I think is not used in the code.

**Comment by marquesfm** [comment]
ok for me

**Pull request merged**